.github/workflows/code-quality.yml

```
@@ -0,0 +1,171 @@
+        with:
+          node-version: '20'
+          cache: 'pnpm'
+
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Run Biome linting (with null safety checks)
+        run: |
+          echo "🔍 Running Biome linting with null safety validation..."
+          pnpm biome:check
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)